src/Serialize.Linq.NetStandard16/Serialize.Linq.NetStandard16.csproj

```diff
@@ -1,18 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netstandard1.6</TargetFramework>
     <RootNamespace>Serialize.Linq</RootNamespace>
-    <Version>1.4.0</Version>
+    <Version>1.4.0$(VersionSuffix)</Version>
     <Company />
-    <Product>Serialize.Linq.NetCore11</Product>
+    <Product>Serialize.Linq.NetStandard16</Product>
     <Authors />
     <Description>C# library to serialize LINQ expressions</Description>
     <Copyright>Copyright © 2012-2017</Copyright>
     <PackageProjectUrl>https://github.com/esskar/Serialize.Linq</PackageProjectUrl>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
```

Comment:
Hey I can put these back, but they break the build on OSX, I'll add a platform condition for windows.